scipy/stats/_stats_py.py

```diff
@@ -8215,8 +8239,8 @@ def combine_pvalues(pvalues, method='fisher', weights=None):
 
     else:
         raise ValueError(
-            "Invalid method '%s'. Options are 'fisher', 'pearson', \
-            'mudholkar_george', 'tippett', 'or 'stouffer'", method)
+            "Invalid method '%s'. Valid methods are 'fisher', 'pearson', \
+            'mudholkar_george', 'tippett', and 'stouffer'", method)
```

Comment:
Since you touch this, let's make it pretty
```suggestion
            f"Invalid method {method!r}. Valid methods are 'fisher', 'pearson', "
            "'mudholkar_george', 'tippett', and 'stouffer'"
        )
```